examples/dynamics/rotating_pendulum.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python3
+
+from sympy import (
+    symbols,
+    Function as SymFunction,
+    cos,
+    sin,
+    solve,
+    Eq,
+    reduce_inequalities,
+)
+from sympy.physics.units import acceleration_due_to_gravity
+from symplyphysics import (
+    Vector,
+    cross_cartesian_vectors,
+    vector_unit,
+    dot_vectors,
+    print_expression,
+)
+from symplyphysics.definitions import (
+    harmonic_oscillator_is_second_derivative_equation as harmonic_def,
+    period_from_angular_frequency as period_def,
+)
+from symplyphysics.laws.dynamics.vector import (
+    relative_acceleration_from_force as motion_law,
+)
+from symplyphysics.laws.kinematic.vector import (
+    acceleration_of_transfer_between_relative_frames as tranfer_law,
+    centripetal_acceleration_via_cross_product as centripetal_law,
+)
+from symplyphysics.laws.kinematic import (
+    tangential_acceleration_of_rotating_body as tangential_law,
+)
+
+# Description
+## A physical pendulum consisting of a ball fixed on the end of a thin rigid rod can freely
+## oscillate around the horizontal axis A passing through the upper end of the rod. Axis A
+## is fixed rigidly on the geometric axis of a horizontal disk uniformly rotating around this
+## (vertical) geometric axis with angular velocity `w`. Thus, the plane of oscillation of the
```

Comment:
Previously you've stated that axis A is horizontal. Is it vertical or horizontal?